app/Resources/CraueConfigBundle/translations/CraueConfigBundle.en.yml

```diff
@@ -3,7 +3,7 @@ download_pictures: Download pictures onto your server
 carrot: Enable share to Carrot
 diaspora_url: diaspora* URL, if the service is enabled
 export_epub: Enable ePub export
-export_mobi: Enable .mobi export
+export_mobi: Enable .mobi export (deprecated, will be removed in 2.6.0)
```

Comment:
Can't this be directly in the template to avoid having all traductions to be up to date?
I put the deprecated message for Baggy directly in the template, without translation.

Reply:
It would be only in english then?

Reply:
Yes I think it's ok because the message will be removed anyway in 2.6.0

Reply:
Fine